receiver/otlpreceiver/config.md

```diff
@@ -5,30 +5,30 @@ Config defines configuration for OTLP receiver.
 
 ### Config
 
-| Name | Type | Default | Docs |
-| ---- | ---- | ------- | ---- |
-| protocols |[otlpreceiver-Protocols](#otlpreceiver-Protocols)| <no value> | Protocols is the configuration for the supported protocols, currently gRPC and HTTP (Proto and JSON).  |
+| Name | Type                                              | Default | Docs |
+| ---- |---------------------------------------------------| ------- | ---- |
```

Comment:
why these changes in a PR that fixes dead links?

Reply:
Goland likes to format tables when you open markdown files in the editor.

Reply:
Would you like me to submit a separate PR that only updates the table formats?

Reply:
Yes please

Reply:
@bogdandrutu [done](https://github.com/open-telemetry/opentelemetry-collector/pull/5743)